### PR TITLE
Feat/calculations

### PR DESCRIPTION
**Pull request by chaitanyapotti**

### CODE REVIEW

File: app/src/content-loader/WalletActivityLoaderMobile.vue [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-  <content-loader :height="600" :width="450" :speed="2" primaryColor="#E7EDF3" secondaryColor="#ecebeb">
-    <rect x="15.5" y="17.61" rx="0" ry="0" width="46.13" height="28.18" />
-    <rect x="15.5" y="63.61" rx="0" ry="0" width="139.18" height="22.12" />
-    <rect x="15.5" y="115.61" rx="0" ry="0" width="418.85" height="72.62" />
-    <rect x="173.5" y="94.61" rx="0" ry="0" width="255.58" height="13.46" />
-    <rect x="15.5" y="195.61" rx="0" ry="0" width="418.85" height="72.62" />
-    <rect x="15.5" y="275.61" rx="0" ry="0" width="418.85" height="72.62" />
-    <rect x="15.5" y="357.61" rx="0" ry="0" width="418.85" height="72.62" />
-  </content-loader>
+  <div class="sk-grid">
```
Comment (iamonuwa) [inline app/src/content-loader/WalletActivityLoaderMobile.vue]: Why did you switch from `vue-content-loader` to normal loader?

File: app/src/views/Confirm/Confirm.vue [hunk 2]
```diff
@@ -470,8 +471,9 @@ export default {
     },
     getCurrencyMultiplier() {
       const { selectedCurrency, currencyData } = this.$store.state || {}
-      let currencyMultiplier = 1
-      if (selectedCurrency !== 'ETH') currencyMultiplier = currencyData[selectedCurrency.toLowerCase()] || 1
+      let currencyMultiplierNum = 1
+      if (selectedCurrency !== 'ETH') currencyMultiplierNum = currencyData[selectedCurrency.toLowerCase()] || 1
+      const currencyMultiplier = new BigNumber(currencyMultiplierNum)
```
Comment (iamonuwa) [inline app/src/views/Confirm/Confirm.vue]: Can you return this as one liner than breaking in 2?

File: app/src/content-loader/WalletActivityLoaderMobile.vue (hunk 1 above)
Comment (chaitanyapotti) [reply app/src/content-loader/WalletActivityLoaderMobile.vue]: i didn't. Lionell merged a PR into this.!

Comment (lionellbriones) [reply app/src/content-loader/WalletActivityLoaderMobile.vue]: > Why did you switch from `vue-content-loader` to normal loader?
its to be consistent with the html loader (we're using this to loader when vue app is not yet available). SVG animation doesn't work when vue app is not loaded yet

File: app/src/views/Confirm/Confirm.vue (hunk 2 above)
Comment (chaitanyapotti) [reply app/src/views/Confirm/Confirm.vue]: Done

**Pull request merged**